presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -1727,9 +1731,14 @@ private void verifyJvmTimeZone()
     private static HiveStorageFormat extractHiveStorageFormat(Table table)
     {
         StorageFormat storageFormat = table.getStorage().getStorageFormat();
-        String outputFormat = storageFormat.getOutputFormat();
+        String outputFormat;
+        try {
+            outputFormat = storageFormat.getOutputFormat();
+        }
+        catch (IllegalStateException e) {
+            throw new PrestoException(HIVE_UNSUPPORTED_FORMAT, format("Illegal output format for table %s: %s", table.getTableName(), e.getMessage()));
```

Comment:
OK. So , what do you think should be the logic of this test case?

Reply:
Are you able to reproduce the case you are facing there?

Reply:
We should use `HIVE_INVALID_METADATA` here. Null means for format is missing, which is different than unsupported.

Reply:
@kokosing 
OK. I am trying to create such a table in test case, but I am not sure this table will be created successfully because of a lot of check and validation;
I am doing this;